contrib/julia-mode.el

```diff
@@ -3095,6 +3095,42 @@ end"))
 (puthash "\\mtteight" "𝟾" julia-latexsubs)
 (puthash "\\mttnine" "𝟿" julia-latexsubs)
 
+;; Code for `inferior-julia-mode'
+(require 'comint)
+
+(defvar julia-file-path "/usr/bin/julia" "Path to the program used by `inferior-julia'.")
```

Comment:
It might be nicer to use `defcustom` here, what do you think? I'd also suggest just using `"julia"` rather than hard-coding the path.